README.md

```diff
@@ -7,7 +7,8 @@ This BaseApp contains dependencies shared between ALL Sugar Labs applications.
 ```
 git clone --recurse-submodules https://github.com/flathub/org.sugarlabs.BaseApp.git
 cd org.sugarlabs.BaseApp
-flatpak -y --user install org.gnome.{Platform,Sdk}//44
+flatpak --user remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
+flatpak --user install flathub-beta org.gnome.{Platform,Sdk}//46beta
```

Comment:
Now that GNOME 46 is released, we should update all these references to the "46" branch, instead of "beta46".